frontend/src/scenes/experiments/Experiment.scss

```diff
@@ -23,7 +23,7 @@
             }
 
             .ant-slider-rail {
-                background-color: var(--primary-highlight);
```

Comment:
Can't we solve this wholesale by setting `--primary: var(--primary-3000);` in `{light,dark}-mode-3000-variables`? @daibhin

Reply:
Good idea. It seems to work too but changes things like this one example I found pretty quickly:
|Before|After|
|----|----|
| <img width="650" alt="Screenshot 2023-11-29 at 13 37 07" src="https://github.com/PostHog/posthog/assets/6685876/8e6c01d9-e726-4277-9c7a-511bf71d9337"> | <img width="657" alt="Screenshot 2023-11-29 at 13 37 27" src="https://github.com/PostHog/posthog/assets/6685876/8e52f00d-a59e-40f9-809f-e54ffe2752dc"> |

I actually think that looks better in the old blue color but happy to change things if you think it's better and we should get rid of all the blue